chapter1/configuration.md

```diff
@@ -1,4 +1,55 @@
-- Webportal login
-- navigators, options
-  (Screen shots)
+## Basic Configuration
+
+### Launch webportal
+In `appliance_console`, we can check either `ManageIQ Server` is running or not. If not then we can start it by entering digit `14` in the advanced setting.  After that when we hit IPv4 address which is given in `appliance_console` in web browser we can see the ManageIQ's Log In page.
+
+![login](../images/chapter1/manageiq_login.png "ManageIQ Log In Page")
+
+### Log In
+There are default credentials for ManageIQ. We can change it but we will discuss it later here.
+
+- **Login:** admin
+- **Password:** smartvm
+
+Now we can see the dashboard which looks like following image
+
+![dashboard](../images/chapter1/manageiq_dashboard.png "ManageIQ Dashboard")
+
+In the extreme left, we can see the vertical navigation bar. We can easily navigate through elements in menu from those bars.
```

Comment:
I think this should be `...elements from menus of this navigation bar.`

Reply:
Yes,thats what I mean to say here but did this accidentally.Thanks again.